client/render-system.cpp

```diff
@@ -449,14 +449,44 @@ namespace tec {
 		for (int i = 0; i < data->entity.components_size(); ++i) {
 			const proto::Component& comp = data->entity.components(i);
 			switch (comp.component_case()) {
-			case proto::Component::kRenderable:
-			{
-				Renderable* renderable = new Renderable();
-				renderable->In(comp);
+				case proto::Component::kRenderable:
+				{
+					Renderable* renderable = new Renderable();
+					renderable->In(comp);
 
-				RenderableMap::Set(entity_id, renderable);
-			}
-			break;
+					RenderableMap::Set(entity_id, renderable);
+				}
+					break;
+				case proto::Component::kPosition:
+					break;
```

Comment:
Most of these breaks can be removed as the cases will falldown till exit regardless.